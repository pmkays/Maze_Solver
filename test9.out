S*******************
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................E
East
East
East
East
East
East
East
East
East
East
East
East
East
East
East
East
East
East
East
South
South
South
South
South
South
South
South
South
South
South
South
South
South
South
South
South
South
South
