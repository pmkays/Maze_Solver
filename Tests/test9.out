S*******************
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................*
...................E
east
east
east
east
east
east
east
east
east
east
east
east
east
east
east
east
east
east
east
south
south
south
south
south
south
south
south
south
south
south
south
south
south
south
south
south
south
south
